javascript/ql/src/semmle/javascript/security/dataflow/IndirectCommandArgument.qll

```diff
@@ -0,0 +1,66 @@
+import javascript
```

Comment:
This module needs a header comment. It also looks like it might not have been autoformatted.